src/ledger/database/rocksdb.zig

```diff
@@ -87,6 +87,9 @@ pub fn RocksDB(comptime column_families: []const ColumnFamily) type {
         pub fn count(self: *Self, comptime cf: ColumnFamily) Allocator.Error!u64 {
             const live_files = try self.db.liveFiles(self.allocator);
             defer live_files.deinit();
+            defer for (live_files.items) |file| {
```

Comment:
Thoughts on single line? 
```suggestion
            defer for (live_files.items) |file| file.deinit();
```

Reply:
9b16c32fc54d6075160bfd48ed1d682d2f481bd7